client/js/lounge.js

```diff
@@ -905,14 +905,11 @@ $(function() {
 						icon: "img/logo-64.png",
 						tag: target
 					});
-					notify.onclick = function() {
+					notify.addEventListener("click", function() {
```

Comment:
I just think registering event listeners as properties on an object is ugly, and one should use the `EventTarget` interface whenever possible.